File: CF/proper.py
import math
mon = int(input())
cola = int(input())
bar = int(input())
x, y, flg, ans1, ans2 = 0, 0, 0, 0, 0
lim = math.floor(mon / min(cola, bar))
if mon / cola != 0:
	for x in range(0, lim + 1):
		for y in range(0, lim + 1):
			if((y * cola) +(x * bar)) == mon:
				ans1 = y
				ans2 = x
				flg = 1
				break
if flg == 1:
	print('YES')
	print(ans1, ans2)
elif flg == 0:
	print('NO')
			

File: CF/proper.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	ios::sync_with_stdio(false);
	long mon = 0, cola = 0, bar = 0;
	cin>>mon;
	cin>>cola;
	cin>>bar;
	for(int i = 0; i * cola <= mon; ++i)
		if((mon - i * cola) % bar == 0)
		{
			cout<<"YES"<<"\n";
			cout<<i<<" "<<(mon - i * cola) / bar<<"\n";
			return 0;
		}
	cout<<"NO"<<"\n";
	return 0;	
	
}
